shared/styles/index.native.js

```diff
@@ -113,6 +117,7 @@ export {
   backgroundURL,
   navBarHeight,
   tabBarHeight,
+  statusBarHeight,
```

Comment:
You should define this in desktop's style too, so that the flow type declaration is truthful